tsconfig.utils.json

```diff
@@ -0,0 +1,7 @@
+{
+	"extends": "./tsconfig.json",
+	"compilerOptions": {
```

Comment:
https://www.typescriptlang.org/tsconfig#declarationMap is a nice option to enable for users.

Reply:
ah yeah! very nice! thanks! :smiley:

Reply:
works great!!! thanks! 😍